scripts/deploy_ephemeral_env.sh

```diff
@@ -0,0 +1,41 @@
+source ${CICD_ROOT}/_common_deploy_logic.sh
+
+# Caller can define any extra deploy arguments to be passed to bonfire
+: ${EXTRA_DEPLOY_ARGS:=""}
+
+# Caller can specify the type of pool to use
+: ${NAMESPACE_POOL:="default"}
+
+# Caller can specify the reservation duration (default: 1h)
+: ${RESERVE_DURATION:="1h"}
+
+# Caller can alter the default dependency fetching method if desired
+: ${OPTIONAL_DEPS_METHOD:="hybrid"}
+
+# Whether or not to deploy frontends (default: false)
+: ${DEPLOY_FRONTENDS:="false"}
+
+# Deploy k8s resources for app and its dependencies (use insights-stage instead of insights-production for now)
+# -> use this PR as the template ref when downloading configurations for this component
+# -> use this PR's newly built image in the deployed configurations
+set -x
+export NAMESPACE=$(bonfire namespace reserve --pool ${NAMESPACE_POOL} --duration ${RESERVE_DURATION})
+SMOKE_NAMESPACE=$NAMESPACE  # track which namespace was used here for 'teardown' in common_deploy_logic
+
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
```

Comment:
Just curious, doesn't the bootstrap script take care of this already?

Reply:
It should, I forget exactly what happened but there was a mismatch on the formatting of the image tag iirc. I'll take another look.

Reply:
Figured out the mismatch: originally the `build_deploy.sh` simply used that `IMAGE_TAG=$(git rev-parse --short=7 HEAD)`, but because this was coming in as a PR the bootstrap script was changing the tag to include the PR number: https://github.com/RedHatInsights/cicd-tools/blob/main/bootstrap.sh#L50

I added the `IMAGE_TAG` to the pipeline environment variables so that it can be used by the `EXTRA_DEPLOY_ARGS` in the ephemeral deploy script to supply the second image.